### DISCUSSION TITLE
blog/technical-analysis/

### DISCUSSION THREAD

**Discussion by giscus** [michaelneuper/website#44] [General]
# blog/technical-analysis/

Systematic vs Discretionary Trading Trading strategies can be classified into two broad categories:
Systematic Trading Systematic trading involves adhering to a predefined set of rules or algorithms to execute trades. These rules are usually based on historical data analysis and quantitative modelling. The advantage of these systems is that you can provably verify your hypothesis / system on real-world data using back-testing / forward-testing and obtain quantifiable metrics to evaluate and build trust in your algorithm.

http://localhost:8000/blog/technical-analysis/

**Comment by akcube** [comment]
test
